docs/eventing/samples/cronjob-source/README.md

```diff
@@ -51,7 +51,7 @@ spec:
   sink:
     apiVersion: serving.knative.dev/v1alpha1
     kind: Service
-    name: message-dumper
+    name: event-display
 ```
 
 Use following command to create the event source from `cronjob-source.yaml`:
```

Comment:
I can't seem to comment on line 69.

The 'Verify' command should be:

```shell
kubectl logs -l serving.knative.dev/service=event-display -c user-container --since=10m
```

The output text will also be a bit different, but I don't know exactly what it will look like right now. We can probably leave the existing text for the time being.